api/routes.py

```diff
@@ -239,6 +239,14 @@ def public_key_document():
 def acquisition_groups(lane_identifier):
     return app.manager.opds_feeds.groups(lane_identifier)
 
+@library_route('/qa-feed')
+@has_library
+@allows_patron_web
+@returns_problem_detail
+@compressible
+def qa_feed():
+    return app.manager.opds_feeds.qa_feed()
+
```

Comment:
This should require patron authorization, since its only purpose is to show books to a QA test script that's going to conduct transactions on the books.